src/main/java/oracle/kubernetes/operator/helpers/ConfigMapHelper.java

```diff
@@ -1,143 +1,165 @@
   }
 
-  @Override
-  public void clear() {
-    throw new UnsupportedOperationException();
-  }
-
-  @Override
-  public Set<String> keySet() {
-    Set<String> keys = new HashSet<>();
-    String[] list = mountPointDir.list();
-    if (list != null) {
-      for (String s : list) {
-        keys.add(s);
-      }
+  private static class ScriptConfigMapStep extends Step {
+    private final String namespace;
+    
+    public ScriptConfigMapStep(String namespace, Step next) {
+      super(next);
+      this.namespace = namespace;
     }
-    return keys;
-  }
-
-  @Override
-  public Collection<String> values() {
-    throw new UnsupportedOperationException();
-  }
-
-  @Override
-  public Set<Entry<String, String>> entrySet() {
-    Set<Entry<String, String>> entries = new HashSet<>();
-    String[] list = mountPointDir.list();
-    if (list != null) {
-      for (String s : list) {
-        entries.add(new Entry<String, String>() {
 
-          @Override
-          public String getKey() {
-            return s;
+    @Override
+    public NextAction apply(Packet packet) {
+      String name = KubernetesConstants.DOMAIN_CONFIG_MAP_NAME;
+      V1ConfigMap cm = new V1ConfigMap();
+      
+      V1ObjectMeta metadata = new V1ObjectMeta();
+      metadata.setName(name);
+      metadata.setNamespace(namespace);
+      
+      AnnotationHelper.annotateWithFormat(metadata);
+      
+      Map<String, String> labels = new HashMap<>();
+      labels.put(LabelConstants.CREATEDBYOPERATOR_LABEL, "true");
+      metadata.setLabels(labels);
+
+      cm.setMetadata(metadata);
+
+      Map<String, String> data = new HashMap<>();
+      
+      data.put("livenessProbe.sh", 
+          "#!/bin/bash\n" + 
+          "# Kubernetes periodically calls this liveness probe script to determine whether\n" + 
+          "# the pod should be restarted. The script checks a WebLogic Server state file which\n" + 
+          "# is updated by the node manager.\n" + 
+          "STATEFILE=/shared/domain/$1/servers/$2/data/nodemanager/$2.state\n" + 
+          "if [ `jps -l | grep -c \" weblogic.NodeManager\"` -eq 0 ]; then\n" + 
+          "  exit 1\n" + 
+          "fi\n" + 
+          "if [ -f ${STATEFILE} ] && [ `grep -c \"FAILED_NOT_RESTARTABLE\" ${STATEFILE}` -eq 1 ]; then\n" + 
+          "  exit 1\n" + 
+          "fi\n" + 
+          "exit 0");
+      
+      data.put("readinessProbe.sh", 
+          "#!/bin/bash\n" + 
+          "\n" + 
+          "# Kubernetes periodically calls this readiness probe script to determine whether\n" + 
+          "# the pod should be included in load balancing. The script checks a WebLogic Server state\n" + 
+          "# file which is updated by the node manager.\n" + 
+          "\n" + 
+          "STATEFILE=/shared/domain/$1/servers/$2/data/nodemanager/$2.state\n" + 
+          "\n" + 
+          "if [ `jps -l | grep -c \" weblogic.NodeManager\"` -eq 0 ]; then\n" + 
+          "  exit 1\n" + 
```

Comment:
The original liveness probe shell script has echos that perhaps should be put back in:

if [ `jps -l | grep -c " weblogic.NodeManager"` -eq 0 ]; then
  echo "Error: WebLogic NodeManager process not found."
  exit 1
fi

if [ -f ${STATEFILE} ] && [ `grep -c "FAILED_NOT_RESTARTABLE" ${STATEFILE}` -eq 1 ]; then
  echo "Error: WebLogic Server FAILED_NOT_RESTARTABLE."
  exit 1
fi

The echo's are actually helpful for debugging as kubernetes records them and reports them when you kubectl describe events...

Ditto for the new readinessProbe script.